README.md

```diff
@@ -1 +1,96 @@
-# Image-Processing-Using-FPGAs
\ No newline at end of file
+# Image-Processing-Using-FPGAs
+
+## ABSTRACT
+
+The aim of this project is to learn the basics of FPGA (its work, functioning, uses) and implement the HDL (Hardware Descriptive Language) learnt for image processing applications. The original task was to perform the following operations on the given grayscale image using FGPA board –
+1. Increase brightness by ‘value'
+
+<img src="https://github.com/OARSS/Image-Processing-Using-FPGAs/blob/main/Images%20and%20Videos/Images/Increase%20Brightness.jpg" alt="Increase brightness" style="height: 431px; width:571px;"/>
+
+2. Decrease brightness by ‘value’
+
+<img src="https://github.com/OARSS/Image-Processing-Using-FPGAs/blob/main/Images%20and%20Videos/Images/Decrease%20Brightness.jpg" alt="Decrease brightness" style="height: 431px; width:571px;"/>
+
+3. Binarize using ‘threshold’
+
+<img src="https://github.com/OARSS/Image-Processing-Using-FPGAs/blob/main/Images%20and%20Videos/Images/Binarise.jpg" alt="Binarise" style="height: 431px; width:571px;"/>
+
+4. Invert image
+
+<img src="https://github.com/OARSS/Image-Processing-Using-FPGAs/blob/main/Images%20and%20Videos/Images/Inversion.jpg" alt="Inversion" style="height: 431px; width:571px;"/>
```

Comment:
it should be in ```<p> </p>``` tags so that images can be aligned to the center. you can take references from previous year's projects' documentation.